src/plugins/kibana_legacy/public/styles/_base.scss

```diff
@@ -147,7 +108,8 @@ input[type='checkbox'],
 .smallest {
   font-size: 0.7em !important;
 }
-
-.text-monospace {
-  font-family: $euiCodeFontFamily;
+select {
+  color: $euiTextColor;
+  background-color: $euiColorEmptyShade;
 }
```

Comment:
Is this `select` styles necessary??? It will target all general `<select>` items. Not sure where it came from.

Reply:
got it from here:
https://github.com/elastic/kibana/blob/24d29a31b8ee8d6eaa05cbd2c255350ef8b47148/src/legacy/ui/public/styles/_legacy/components/_input.scss#L9
migrated it because I couldn't rule out that it's needed somewhere

Reply:
I can't find this anymore, you deleted it, yes?

Reply:
yes, I checked and it was even overwritten in timelion which is the most legacy plugin, so I think it's safe to remove